### ISSUE TITLE
python setup doom loop

### ISSUE THREAD

**Issue by aeggers** [tsrobinson/rMIDAS#5]
This is a weird one -- maybe an RStudio interaction issue? 

I have a python installation, but not at usr/bin/python. 

I ran `rMIDAS::mid_py_setup()`. 

This part seems okay seems okay: 

> rMIDAS::mid_py_setup()

`The following packages need to be installed:  pandas   tensorflow==1.15   sklearn`  
`Are you happy to proceed? [Y/N]: Y`

`Installing missing python dependency:  pandasError: .onLoad failed in loadNamespace() for 'rMIDAS', details:`
  `call: NULL`
 `error: could not find a Python environment for /usr/bin/python`

OK, an error, fine -- but then it's still trying and apparently hanging:  

`The following packages need to be installed:  pandas   tensorflow==1.15   sklearn`  
`Are you happy to proceed? [Y/N]: Y`

`Installing missing python dependency:  pandas`

I type N to try to stop it, and somehow this goes to the prompt immediately:

> N
`Error: object 'N' not found`

but it's still trying: 

`The following packages need to be installed:  pandas   tensorflow==1.15   sklearn`  
`Are you happy to proceed? [Y/N]: N`

still hanging -- type "N" again, goes to prompt, this time returns focus to prompt

> N
`Error: object 'N' not found`

OK assign 10 to x
> x = 10

And again I get the package installation message: 

`The following packages need to be installed:  pandas   tensorflow==1.15   sklearn`  
`Are you happy to proceed? [Y/N]: N`
> N
`Error: object 'N' not found`

Eventually I get control back, but then if I type `rMIDAS::` it returns to trying to install things.  Like, as soon as I type the second colon it goes back into: 

`The following packages need to be installed:  pandas   tensorflow==1.15   sklearn`  
`Are you happy to proceed? [Y/N]:`

**Comment by tsrobinson** [comment]
Hi @aeggers, I've reijgged the python configuration tools and loading so no more doom loop! This should now work better for non-standard python locations, and we've added a vignette on how to configure this. TLDR: `set_python_env("path/to/binary")` or use virtual env `set_python_env("virtual_env", type = "virtualenv")`. 

No need to call any other functions afterwards which is a plus! 

All packaged and sent to CRAN, but in the meantime feel free to github install.
